shelf/logger_creator.py

```diff
@@ -0,0 +1,108 @@
+import logging
+import sys
+
+
+class LoggerCreator(object):
```

Comment:
Shoot, it looks like this is a tiny bit more complicated than I thought. The problem is `LoggerCreator` would need to support another handler.

I would just add a method to set the handler (like I had it the first run through) but the problem is it makes for a rocky interface since if that is not called first it will make other methods blow.

I guess I am going to leave this for now as per our prime directive.